### DISCUSSION TITLE
Updating to node 16.7 broke my bot.

### DISCUSSION THREAD

**Discussion by vol1n** [PrismarineJS/mineflayer#2132] [General]
Anyone know anything about this?

**Comment by u9g** [comment]
can you give more info?

**Reply by vol1n** [reply]
I was on nodejs 14.7, and then I tried updating to 16.7 for discord.js compatibility. After this, whenever I run a mineflayer script, it does nothing. I have restarted my pc, reinstalled Node 14.7 after clearing every single node file. As you can see there is no error message and no bot joins the server. 
![Screenshot 2021-08-22 164048](https://user-images.githubusercontent.com/67127243/130363175-16a9c1c4-2a5e-4cee-bdbd-6468502095bd.jpg)

**Reply by u9g** [reply]
can you  run the bot using: 
```js
$env:DEBUG="minecraft-protocol";node script.js
```
and tell me what is outputted?

**Reply by vol1n** [reply]
I'm on Windows cmd, it said 
![Screenshot 2021-08-22 164924](https://user-images.githubusercontent.com/67127243/130363320-91a02090-1307-4ebc-ae7b-74630e8616f7.png)

**Reply by u9g** [reply]
can you try in powershell what I said?

**Reply by vol1n** [reply]
![Screenshot 2021-08-22 165449](https://user-images.githubusercontent.com/67127243/130363482-a4b70c93-7d51-4ef8-b8de-47ab6d439ef8.png)

**Reply by u9g** [reply]
okay, can you share the start of your code so maybe I can see why it's not starting?

**Reply by vol1n** [reply]
The code used to work, it's not been changed, so I doubt that is the issue. I can run any js using node unless it uses mineflayer then it just does nothing. 
Here it is:
![Screenshot 2021-08-22 170027](https://user-images.githubusercontent.com/67127243/130363623-d252d21c-72d8-4346-900a-959eee8334e5.png)
